hangman.rb

```diff
@@ -0,0 +1,31 @@
+random_words = ["camera", "photo", "picture", "landscape", "light", "shadow", "lens", "portrait"]
+#the number is one more
+random_num = rand(0..7).to_i
```

Comment:
Ahh, I see, 7 is the number of words in the array. You should actually use `random_words.length`. This way you don't need to keep the number of words in the array synced with this number.

The 7 is what we called hardcoding. It's good for testing purposes, but not with real code. So if I wanted to make sure my app worked. I would always hardcode it at 0 ("camera") so I can keep guessing around it.